chart/values-openshift.yaml

```diff
@@ -0,0 +1,311 @@
+promtail:
+  containerSecurityContext:
+    privileged: true
+    allowPrivilegeEscalation: true
+  serviceAccount:
+    create: false
+    # Service account name must match service account created below
+    name: 'dnation-monitoring-promtail'
+dnation-kubernetes-monitoring:
```

Comment:
Just cosmetic, but please add a space above `dnation-kubernetes-monitoring` to be more readable and consistence in whole file.  Also please do it for `loki`, `thanos`, `extraServices` and `openshift`.